src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1431,15 +1431,21 @@ public void postParse(int type, InputFile f) {
 				case "mov":
 					mimeType = HTTPResource.MOV_TYPEMIME;
 					break;
+				case FormatConfiguration.M4A:
+					mimeType = HTTPResource.AUDIO_M4A_TYPEMIME;
+					break;
+				case FormatConfiguration.AAC:
+					mimeType = HTTPResource.AUDIO_AAC_TYPEMIME;
+					break;
+				case FormatConfiguration.AAC_HE:
+					mimeType = HTTPResource.AUDIO_AAC_TYPEMIME;
+					break;
 				case FormatConfiguration.ADPCM:
 					mimeType = HTTPResource.AUDIO_ADPCM_TYPEMIME;
 					break;
 				case FormatConfiguration.ADTS:
 					mimeType = HTTPResource.AUDIO_ADTS_TYPEMIME;
 					break;
-				case FormatConfiguration.M4A:
-					mimeType = HTTPResource.AUDIO_M4A_TYPEMIME;
-					break;
```

Comment:
I did readd, as when i tried without the AAC trick i get issue.
But it seem that now it's don't affect anything, and sadly not the only 1 file that don't display nicely in the log. I know, it's only esthetical, but i would like understand why...